PictureApp/Controller/OptionsViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  OptionsViewController.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 26.11.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+protocol OptionsViewControllerDelegate : AnyObject {
+    func optionsViewController (_ controller: OptionsViewController, didSelectOrderStrategy order: Order)
+    func didFilterSelected()
+    
+}
+ 
+enum Order :String{
+
+  case popular = "popular"
+  case latest = "latest"
+
+}
+
+class OptionsViewController: UIViewController {
+    
+    @IBOutlet var sortButton: UIButton!
+    var isTapped :Bool = true
+    weak var delegate: PicturesViewController?
+    
+    
+    @IBAction func sortPictures() {
+        
+        self.dismiss (animated: true, completion: nil)
```

Comment:
Better to call dismiss after all work done,  move this to the bottom of the method